multi-datacenter/docker-compose.yml

```diff
@@ -115,6 +115,8 @@ services:
     ports:
       - "8382:8382"
       - "9892:9892"
+    volumes:
+      - mi3:/etc/kafka-connect/jars/
```

Comment:
@thomaskwscott this container's Docker image is already using `confluentinc/cp-enterprise-replicator`, so it's not clear why the PR uses `replicator-for-jar-transfer`?  (same with the connect container below)

Reply:
We need to put the rest extension jar on the Connect worker classpath. All the images hardcode the classpath here: https://github.com/confluentinc/cp-docker-images/blob/5.3.1-post/debian/enterprise-replicator-executable/include/etc/confluent/docker/launch#L41 
There's no easy workaround to move the Replicator jar into the classpath directory when the container starts up as it has to be done before the connect worker is launched. 

The mount solution remounts, just the rest extension jar in the classpath dir (we can't have all of Replicator in there)

Reply:
@thomaskwscott we should avoid this extra Docker container and instead implement a cleaner solution:

1. Why wouldn't this work in the same manner as https://github.com/confluentinc/cp-demo/blob/5.3.1-post/docker-compose.yml#L169

2. If it doesn't work, a workaround: copy the jar to the file and then let the Docker container continue (e.g. example https://github.com/confluentinc/cp-demo/blob/5.3.1-post/docker-compose.yml#L32)

Reply:
I trued with the approach from 1 initially, that's how I found the hard coded value. I suspect this is not doing anything in cp-demo and won't work here. 
The 2 approach is nice, will try it today.

Reply:
@thomaskwscott IIUC, you are saying that this line is a noop: https://github.com/confluentinc/cp-demo/blob/5.3.1-post/docker-compose.yml#L169 ?

That doesn't appear to be the case.  Here's a test, I changed CLASSPATH to `foobar`, and the result is it gets set in the Docker container:

```
$ docker-compose exec connect env | grep CLASSPATH
CLASSPATH=foobar
CUB_CLASSPATH=/etc/confluent/docker/docker-utils.jar
```

So can you try by setting the `CLASSPATH` to include the required jar?  That would be a simpler method than overriding the `command` statement.

Reply:
As we discussed in Slack:

1. cp-demo uses that ENV CLASSPATH setting for running kafka-console-producer/consumer commands from CLI
2. @thomaskwscott will raise a jira to augment the Docker image to allow CLASSPATH to be overridden so we don't have to do the command hack (please post jira here)